### DISCUSSION TITLE
Way to see all classes of a package from a 3rd party library

### DISCUSSION THREAD

**Discussion by yadavan88** [scalameta/metals#4559] [Q&A] [answered]
Hi,
I want to see the package structure of a package(of a 3rd party dependency) when clicked on it in vs code metals. I used to do this in IntelliJ a lot to understand and explore different classes of a package/library. However, in metals, I am not able to find this feature. 
I am attaching the screenshot from IntelliJ IDEA for clarity. Is this possible in vscode? 

<img width="865" alt="image" src="https://user-images.githubusercontent.com/9131956/197327705-26288cbe-7cb2-4259-9578-6e7e360545b0.png">

**Comment by ckipp01** [comment, marked as answer]
Hey @yadavan88, yes we have a very similar feature that we call the tree view. You can see this if you click the Metals logo on the left hand side of VS Code and then go under "Packages". I've included a snapshot below:

<img width="1266" alt="Screenshot 2022-10-22 at 10 23 17" src="https://user-images.githubusercontent.com/13974112/197328945-817d39f1-9ce7-47ff-a44a-27c33ab6571a.png">

**Reply by yadavan88** [reply]
Thank you very much for quick reply. Yes, i was looking for this feature. I was looking in all other places except that section.
Also, I take this opportunity to ask one more question. I able to view the library classes, but except for scala3-library. It shows empty classes. Is this a bug? Please see the screenshot. It shows no classes under reflect or compiletime orderiving packages. But shows under runtime.function package.
![image](https://user-images.githubusercontent.com/9131956/197329966-ac52a95d-ca8c-4ffe-9e3b-11c8319bf06e.png)

**Reply by ckipp01** [reply]
This feature isn't yet implemented for Scala 3.

**Reply by yadavan88** [reply]
Thanks again!
